content/version/2/0/code_of_conduct.de.md

```diff
@@ -0,0 +1,137 @@
++++
+version = "2.0"
+aliases = ["/version/2/0/de"]
++++
+
+# Vereinbarung über Verhaltenskodex für Mitwirkende
+
+## Unsere Verpflichtung
+
+Wir als Mitglieder, Teilnehmer und Verantwortliche unserer Gemeinschaft
```

Comment:
I would suggest to use "Teilnehmende" instead of "Teilnehmer" because it's more gender neutral.